src/System.IO.FileSystem/tests/PortedCommon/IOInputs.cs

```diff
@@ -14,7 +14,8 @@ internal static class IOInputs
         new char[] { '\"', '<', '>', '|', '\0', (Char)1, (Char)2, (Char)3, (Char)4, (Char)5, (Char)6, (Char)7, (Char)8, (Char)9, (Char)10, (Char)11, (Char)12, (Char)13, (Char)14, (Char)15, (Char)16, (Char)17, (Char)18, (Char)19, (Char)20, (Char)21, (Char)22, (Char)23, (Char)24, (Char)25, (Char)26, (Char)27, (Char)28, (Char)29, (Char)30, (Char)31, ':', '*', '?' } :
         new char[] { '\0' };
 
-    public static bool SupportsCreationTime { get { return RuntimeInformation.IsOSPlatform(OSPlatform.Windows) | RuntimeInformation.IsOSPlatform(OSPlatform.OSX); } }
+    public static bool SupportsSettingCreationTime { get { return RuntimeInformation.IsOSPlatform(OSPlatform.Windows); } }
```

Comment:
I mean should we consider having public Syste.IO.File.CanSetCreationTime, System.IO.Pipe.CanSetCapacity, etc. so that portable code doesn't have to catch PlatformNotSupported nor hard-code OS checks.

Just something to consider as a future feature and was wondering if we already had an issue or issue(s) about it...